libs/blocks/caas/utils.js

```diff
@@ -234,6 +234,7 @@ export const getConfig = async (state, strs = {}) => {
       },
       setCardBorders: state.setCardBorders,
       useOverlayLinks: state.useOverlayLinks,
+      collectionButtonStyle: state.collectionBtnStyle,
```

Comment:
@cmiqueo, I think you need to define a `defaultState` for this (found at the bottom of the file). Otherwise the component may not initialize correctly. 

Also this change may require a change also in a corresponding unit tests (since there is a test that checks config deeply equals some pre-defined value). Otherwise that test would fail.

Reply:
Done